src/Simplex/Chat.hs

```diff
@@ -314,10 +323,23 @@ withStore action = do
 chatCommandP :: Parser ChatCommand
 chatCommandP =
   ("/help" <|> "/h") $> ChatHelp
+    <|> ("/group #" <|> "/g #") *> (NewGroup <$> groupRef)
+    <|> ("/add #" <|> "/a #") *> (AddMember <$> groupRef <* A.space <*> contactRef <* A.space <*> memberRole)
+    <|> ("/remove #" <|> "/rm #") *> (RemoveMember <$> groupRef <* A.space <*> contactRef)
+    <|> ("/delete #" <|> "/d #") *> (DeleteGroup <$> groupRef)
+    <|> ("/members #" <|> "/ms #") *> (ListMembers <$> groupRef)
```

Comment:
not a critique, just a thought - maybe we could have a subcommand for group commands (e.g. instead of "#" - `/group new`, `/group add`, `/group remove`, etc.), so that it's harder to mix up groups and regular contacts namespaces? also maybe move group ChatCommands under GroupCommand which is also a ChatCommand but has different constructors of its own to somewhat separate group commands on types level as well? I'm not sure of benefits other than clearer distinction between group and contact commands on types and parsers, but I just felt it starts to fill somewhat crowded in ChatCommands and probably will only become more so. For example if we later introduce something like "Service" or "Configuration commands" letting client to control app - they'd also be mixed into the same ChatCommands namespace. Instead we could have separate namespaces for them, groups, actually regular contacts as well, and maybe others

Reply:
will think about it... not needed at the moment